app/javascript/react/screens/App/Overview/screens/PlanWizard/components/PlanWizardVMStep/PlanWizardVMStepConstants.js

```diff
@@ -7,5 +7,7 @@ export const V2V_VM_POST_VALIDATION_REASONS = {
   in_other_plan: __('VM exists in a different migration plan'),
   migrated: __('VM has already been migrated'),
   not_exist: __('VM does not exist'),
-  ok: __('VM available for migration')
+  ok: __('VM available for migration'),
+  duplicate: __('Duplicate VM'),
+  no_info_available: __('No information available')
```

Comment:
In the case of no information, the vm in question is unavailable for migration correct?  If so, we might want to add a few words, to indicate so maaaaybeeeeeeee something similar to `VM unavailable for migration, no information available`

Reply:
Done in 9774ad0